pkg/surveyext/editor_manual.go

```diff
@@ -87,7 +87,7 @@ func edit(editorCommand, fn, initialValue string, stdin io.Reader, stdout io.Wri
 	cmd := exec.Command(args[0], args[1:]...)
 	cmd.Env = env
 	cmd.Stdin = stdin
-	cmd.Stdout = stdout
+	cmd.Stdout = os.Stdout
```

Comment:
Seems like this is a non-related change?

Reply:
Good catch, thanks!